new-standalone-service.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+cleanup() {
+    # TODO(mohit): Maybe add a trap for SIGINT.
+    rm -rf template.zip
+}
+
+info() {
+    MESSAGE=$1
+    echo $MESSAGE
+}
+
+debug() {
+    if [[ -z "${DEBUG// }" ]]; then
+        return
+    fi
+    MESSAGE=$1
+    echo "DEBUG: $MESSAGE"
+}
+
+PROJECT_NAME=$1
+PROJECT_PATH=$2
+
+if [[ -z "${PROJECT_NAME// }" ]]; then
+    echo "You must provide the name of the project as the first argument"
+    echo "Usage: ./new-standalone-service.sh project-name"
+    echo "Example: ./new-standalone-service.sh kafka"
+    cleanup
+    exit 1
+fi
+
+if [[ -z "${PROJECT_PATH// }" ]]; then
+    PROJECT_PATH=$(pwd)
+fi
+
+debug "Scaffolding $PROJECT_NAME from template"
+
+cp -R frameworks/template $PROJECT_PATH/$PROJECT_NAME
+cp -R tools $PROJECT_PATH/$PROJECT_NAME/tools
+
+rm -rf $PROJECT_PATH/$PROJECT_NAME/build
+rm -rf $PROJECT_PATH/$PROJECT_NAME/cli/dcos-*/*.whl
+rm -rf $PROJECT_PATH/$PROJECT_NAME/cli/dcos-*/dcos-*
+rm -rf $PROJECT_PATH/$PROJECT_NAME/cli/python/{build,dist}
+rm -rf $PROJECT_PATH/$PROJECT_NAME/build.sh
+
+cat > $PROJECT_PATH/$PROJECT_NAME/build.sh <<'EOF'
+#!/bin/bash
+set -e
+
+# capture anonymous metrics for reporting
+curl https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/sdk/build-sh-start.png >/dev/null 2>&1
+
+FRAMEWORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+BUILD_DIR=$FRAMEWORK_DIR/build/distributions
+PUBLISH_STEP=${1-none}
+export REPO_NAME="$(basename $FRAMEWORK_DIR)"
+export BUILD_BOOTSTRAP=no
+export TOOLS_DIR=${FRAMEWORK_DIR}/tools
+export CLI_DIR=${FRAMEWORK_DIR}/cli
+export ORG_PATH=github.com/$REPO_NAME
+${FRAMEWORK_DIR}/tools/build_framework.sh $PUBLISH_STEP $REPO_NAME $FRAMEWORK_DIR $BUILD_DIR/$REPO_NAME-scheduler.zip
+
+# capture anonymous metrics for reporting
+curl https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/sdk/build-sh-finish.png >/dev/null 2>&1
+
+EOF
+
+chmod +x $PROJECT_PATH/$PROJECT_NAME/build.sh
+mv $PROJECT_PATH/$PROJECT_NAME/cli/dcos-template $PROJECT_PATH/$PROJECT_NAME/cli/dcos-$PROJECT_NAME
+mv $PROJECT_PATH/$PROJECT_NAME/src/main/java/com/mesosphere/sdk/template/ $PROJECT_PATH/$PROJECT_NAME/src/main/java/com/mesosphere/sdk/$PROJECT_NAME/
+mv $PROJECT_PATH/$PROJECT_NAME/src/test/java/com/mesosphere/sdk/template/ $PROJECT_PATH/$PROJECT_NAME/src/test/java/com/mesosphere/sdk/$PROJECT_NAME/
+
+if [ $? -ne 0 ]; then
+    echo "Failed to create new project"
+    cleanup
+    exit 1
+fi
+
+UPPER_CASE_PROJECT_NAME=$(echo $PROJECT_NAME | awk '{print toupper($0)}')
+
+find $PROJECT_PATH/$PROJECT_NAME -type f -exec sed -i.bak "s/template/$PROJECT_NAME/g; s/TEMPLATE/$UPPER_CASE_PROJECT_NAME/g; s/template/$PROJECT_NAME/g" {} \;
+sed -i.bak -e '21,$ d' $PROJECT_PATH/$PROJECT_NAME/src/main/dist/svc.yml
+find $PROJECT_PATH/$PROJECT_NAME -type f -name *.bak -exec rm {} \;
```

Comment:
I hit this locally:
```
rm: remove write-protected regular file '/home/nick/code/dcos-commons/dse/tools/shakedown_env/src/shakedown/.git/objects/a2/59f08996c35899ce8205aae51f7f0052682ae7.bak'? y
rm: remove write-protected regular file '/home/nick/code/dcos-commons/dse/tools/shakedown_env/src/shakedown/.git/objects/a2/9e4d25d57dc360038d81ec0175a5eb46509958.bak'? y
[...]
```

To get around that, this command could be written with `rm -f`: `find $PROJECT_PATH/$PROJECT_NAME -type f -name *.bak -exec rm -f {} \;`